FRAuth/FRAuth/AppIntegrity/FRAppIntegrityKeys.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FRAppAuthKeys.swift
+//  FRAuth
+//
+//  Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+
+public struct FRAppIntegrityKeys {
```

Comment:
Missing doc in all public methods and attributes

Reply:
sorry i totally missed this for whole file..just added